src/main/java/com/Application.java

```diff
@@ -0,0 +1,32 @@
+package com;
```

Comment:
let's remove `core.basesyntax` packages from both `main.java` and `test.java` 
and move our whole project to `com.internet.shop` package inside `main.java` folder

I expect to see the removed `HelloWorld` and `HelloWorldTest` in this PR